kubetest2/internal/deployers/eksapi/templates/unmanaged-nodegroup.yaml.template

```diff
@@ -73,10 +55,6 @@ Resources:
 
   NodeGroup:
     Type: AWS::AutoScaling::AutoScalingGroup
-    UpdatePolicy:
-      AutoScalingRollingUpdate:
-        WaitOnResourceSignals: true
```

Comment:
This `cfn-signal` thing isn't really necessary for our purposes, and it's tricky to merge it with the `UserData` parameter and still have `${AWS::StackName}` populated.